Project.toml

```
@@ -7,7 +7,7 @@ Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 JuliaDB = "a93385a2-3734-596a-9a66-3cfbb77141e6"
 
 [compat]
-JuliaDB = "^0.12"
+JuliaDB = "^0.13"
```

```suggestion
JuliaDB = "^0.12"
```
Let's not bother doing this. I have a PR to phase out JuliaDB.